include/lz4_stream.h

```diff
@@ -54,6 +54,26 @@ class basic_ostream : public std::ostream
   }
 
  private:
+  /**
+   * @brief RAII version of `LZ4F_compressionContext_t`.
+   */
+  struct CompressionContext
```

Comment:
Minor thing. The other classes use snake_case naming. I don't have any strong feelings one way or the other, but I really like to be consistent.

Reply:
Done.